ansible/library/exabgp.py

```diff
@@ -203,14 +206,14 @@ def remove_exabgp_supervisord_conf(name):
     except Exception:
         pass
 
-def setup_exabgp_processor():
+def setup_exabgp_processor(default_dump_script):
     try:
         os.mkdir("/usr/share/exabgp", 0755)
     except OSError:
         pass
-
-    with open("/usr/share/exabgp/dump.py", 'w') as out_file:
-        out_file.write(dump_py)
+    if default_dump_script:
+        with open(DEFAULT_DUMP_SCRIPT, 'w') as out_file:
```

Comment:
Should we use 'default_dump_script' as an argument here?

Reply:
The ```default_dump_script``` is a bool, but the name is confusing. I rename it to ```use_default_dump_script```, I think should be better.